source/docs/quiche_integration.md

```diff
@@ -24,7 +24,7 @@ When upstream or any L7 filter reaches its buffer limit, it will call Http::Stre
 
 #### Send buffer
 
-We use the unlimited stream send buffer in QUICHE along with a book keeping data structure `EnvoyQuicSimulatedWatermarkBuffer` to serve the function of WatermarkBuffer in Envoy to prevent buffering too much in QUICHE.
+We use the unlimited stream send buffer in QUICHE along with a book keeping data structure `EnvoyQuicSimulatedWatermarkBuffer` to serve the function of WatermarkBuffer in Envoy to prevent buffering too much in QUICHE. The QUICHE send send buffer stores bytes to be sent and bytes in flight. The former contributes to `EnvoyQuicSimulatedWatermarkBuffer`, and the latter contributes to congestion control window. The total memory is bound by the high watermark of the connection and max congestion control window.
```

Comment:
Quiche send send -> Quiche send
is bound -> is bounded

Reply:
done